diff --git a/Tables/dbo.Customer.sql b/Tables/dbo.Customer.sql
index bb71d1e..731ee65 100644
--- a/Tables/dbo.Customer.sql
+++ b/Tables/dbo.Customer.sql
@@ -3,7 +3,8 @@
 [CustomerId] [int] NOT NULL,
 [FirstName] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
 [MiddleName] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
-[LastName] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL
+[LastName] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[NewColumn] [int] NOT NULL
 ) ON [PRIMARY]
 GO
 ALTER TABLE [dbo].[Customer] ADD CONSTRAINT [PK_dbo_Customer] PRIMARY KEY CLUSTERED  ([CustomerId]) ON [PRIMARY]
